### ISSUE TITLE
USB hotplug support?

### ISSUE THREAD

**Issue by johalun** [FrodeSolheim/fs-uae#225]
Would it be a lot of work to support this and not having to restart FS-UAE to for example switch out a mouse for a gamepad, or just plugging in gamepads? I'm usually running FS-UAE on a laptop and don't always have all my USB devices plugged in.

**Comment by FrodeSolheim** [comment]
Yes, already implemented for gamepad controllers for the upcoming FS-UAE 4.0 :)

**Comment by johalun** [comment]
Is it possible to test the 4.0 dev branch yet? If so, is it the master branch?

**Comment by FrodeSolheim** [comment]
It is the master branch, but the branch on github isn't completely up to date, and the state right now is such that I recommend avoiding the testing it out for a couple of weeks so I can get the basics into good shape. Read more about the status/progress here: http://eab.abime.net/showthread.php?t=100873 :)

**Comment by FrodeSolheim** [comment]
The master branch is in a much better state now, so it is definitively possible to test this if you don't mind compiling your own version. Won't be that long before official dev builds start appearing either.
